### DISCUSSION TITLE
As of now missing functionality - encrypting folders

### DISCUSSION THREAD

**Discussion by trm217** [FiloSottile/age#585] [General]
I stumbled upon age through hackernews and it looks very much like the crypto library that I've been looking for quite some time.
One thing that I instantly stumbled upon, was the missing option to encrypt / decrypt a folder.
In the past I wrote a library similar to this, where I, when encrypting folders, created an archive automatically if the input file was a folder.

Is there a reason, why encrypting folders is not yet available?

**Comment by FiloSottile** [comment]
age encrypts a single payload, it doesn't define an archive format to turn multiple files into one. There are many good options for that, such as zip and tar, both of which are easy to then encrypt with age. There are even tools like rage-mount that work specifically with age-encrypted zips.
